lib/std/heap/sbrk_allocator.zig

```diff
@@ -39,13 +40,13 @@ pub fn SbrkAllocator(comptime sbrk: *const fn (n: usize) usize) type {
 
         // TODO don't do the naive locking strategy
         var lock: std.Thread.Mutex = .{};
-        fn alloc(ctx: *anyopaque, len: usize, log2_align: u8, return_address: usize) ?[*]u8 {
+        fn alloc(ctx: *anyopaque, len: usize, log2_align: mem.Alignment, return_address: usize) ?[*]u8 {
```

Comment:
```suggestion
        fn alloc(ctx: *anyopaque, len: usize, alignment: mem.Alignment, return_address: usize) ?[*]u8 {
```

Alignment is no longer provided as log2 values

Reply:
(applies to all functions, not just this one)